test/tune/test_flaml_raytune_consistency.py

```diff
@@ -0,0 +1,96 @@
+# import unittest
+import numpy as np
+# require: pip install flaml[blendsearch, ray]
+# require: pip install flaml[ray]
+import time
+from flaml import tune
+from ray import tune as raytune
```

Comment:
add try-except because not all tests are equipped with ray installation. See test_tune.py for example.